addons/website/static/src/snippets/s_dynamic_snippet/000.scss

```diff
@@ -11,4 +11,42 @@
     img {
         object-fit: scale-down;
     }
+
+    //
+    // Handle adaptations according to the container size and o_dynamic_title position.
+    //
+
+    // Define entries minimum size as half 'o_small_container' width.
+    $-container-small-inner-width-max: map-get($container-max-widths, md) - $grid-gutter-width;
+    $-entry-min-width: ($-container-small-inner-width-max * 0.5) - $grid-gutter-width;
+
+    > .container {
+        .o_dynamic_title.o_dynamic_title_aside + .o_dynamic_content .s_dynamic_snippet_row > * {
+            @include media-breakpoint-up(lg) {
+                // Switch to a fluid layout in order to keep entries readable.
+                // Our target grid is 2columns based
+                flex: 0 1 MAX($-entry-min-width, 50%);
```

Comment:
This is uppercase on purpose in order to trigger the CSS function rather than the SASS one. 
(Sass is case sensitive, while css is not.)

_note:  Replying 'cause some changes (like this one) are mine, see https://github.com/odoo/odoo/pull/135509#pullrequestreview-1785499230_ 😉

Reply:
Oh ok, I would have imagined that the scss would be smarter than that and since those use different units (that the max SCSS function can thus not handle), it would fallback on the CSS function automatically (why the hell is this not the case? Are you sure it is not the case? :joy:). If not, I probably would have written it that way then:
```scss
flex: 0 1 max(#{$-entry-min-width}, 50%);
```
But if you don't agree, that's ok, this can stay this way.

Reply:
> why the hell is this not the case? Are you sure it is not the case? 😂

Quite sure 😅
```console
"Internal Error: Incompatible units: 'px' and '%'.
```
Apparently saas introduced match function [way before CSS](https://sass-lang.com/documentation/values/calculations/#legacy-global-functions), and the back-compatibility logic doesn't cover the "incompatible units" scenario.

> But if you don't agree, that's ok, this can stay this way.

Yes, I'd like to keep it this way, thanks  👍